tailscale/DOCS.md

```diff
@@ -121,6 +122,49 @@ When not set, this option is enabled by default.
 
 Received files are stored in the `/share/taildrop` directory.
 
+### Option: `proxy`
+
+When not set, this option is enabled by default.
+
+Tailscale can provide a TLS certificate for your Home Assistant device within
+your tailnet domain.
+
+This can prevent browsers to warn that HTTP URLs to your Home Assistant device
+look unencrypted (browsers are not aware of that connections between Tailscale
+nodes are secured with end-to-end encryption).
```

Comment:
```suggestion
This can prevent browsers from warning that HTTP URLs to your Home Assistant device
look unencrypted (browsers are not aware of the connections between Tailscale
nodes are secured with end-to-end encryption).
```